keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,91 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding box tensor with -1s.
+
+       This is done to convert RaggedTensors to be converted into
```

Comment:
there are some indentation issues across all of the docstrings.  Make sure to match the python code indentation in your docstrings.